src/main/java/com/reckue/post/models/Post.java

```diff
@@ -24,7 +23,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Document
-public class Post {
+public class Post implements Cloneable {
```

Comment:
Don't use Cloneable. Use constructor of copy or **BeanUtils.cloneBean** from Apache Commons:
```
BeanUtils.cloneBean(original)
```